lib/chef/policy_builder/policyfile.rb

```diff
@@ -301,7 +285,10 @@ def cookbook_lock_for(cookbook_name)
       end
 
       # @api private
+      # @return [Array<String>]
       def run_list
+        return override_runlist.map(&:to_s) if override_runlist
```

Comment:
With this in place, do you need to parameterize `run_list_with_versions_for_display`?